gst/datarepo/gstdatareposrc.c

```diff
@@ -138,14 +189,14 @@ gst_data_repo_src_init (GstDataRepoSrc * src)
 {
   src->filename = NULL;
   src->fd = 0;
+  src->media_type = _NNS_TENSOR;
   src->offset = 0;
-  src->read_position = 0;
+  src->successful_read = FALSE;
 
-  /* for test */
-  src->length = 3176;           /* Calculation is required using property, 3176 is MNIST size */
-  //src->item_size[0] = 3136; /* Calculation is required using property */
-  //src->item_size[1] = 40; /* Calculation is required using property */
-  src->item_size[0] = 3176;
+/* let's set value by property */
```

Comment:
minor indent mistake:
```suggestion
  /* let's set value by property */
```

Reply:
Thank you.
This was removed from a recent PR.